src/Components/FollowButton/Button.tsx

```diff
@@ -33,6 +33,7 @@ export class FollowButton extends React.Component<Props, State> {
       : "Follow"
 
     const props = {
+      variant: isFollowed ? "secondaryOutline" : "primaryBlack",
```

Comment:
The order here is important, merging in `...buttonProps` after this means you can still overwrite this to have `secondaryOutline` always (as we do in `ArtistCard`). But by default this will be the behavior.